src/App/App.js

```diff
@@ -50,25 +52,27 @@ class App extends Component {
         <main className='App'>
           <Switch>
             <Route exact path='/' render={() => {
-              if(this.state.errorMessage) {
-                return(<Errors errorMessage={this.state.errorMessage} />)
-              }
-              return (
+              return this.state.errorMessage ? (
+                <Errors errorMessage={this.state.errorMessage} /> 
+              ) : (
                 <>
                   <Search filterPlaces={this.filterPlaces} />
                   <div className='results-component-container'>
-                    <Results places={this.state.results} />
-                  </div>;
+                    {this.state.loading ? (
+                    <p className='loading-message'>Loading your favorite places...</p> 
```

Comment:
Thanks for catching the loading bug. I appreciate the addition of this `loading` state and displayed message! If I'm following this code correctly, it looks like we have nested ternary operations here now? That feels like bad practice, similar to having nested If statements. Let's consider refactoring this differently.